libraries/standard/http/httpFilePaths.cmake

```diff
@@ -18,7 +17,7 @@ set( HTTP_INCLUDE_PUBLIC_DIRS
 # HTTP library Private Include directories.
 set( HTTP_INCLUDE_PRIVATE_DIRS
      ${MODULES_DIR}/standard/http/src
-     ${MODULES_DIR}/standard/http/third_party )
+     ${MODULES_DIR}/standard/http/third_party/http_parser )
```

Comment:
${MODULES_DIR}/standard/http/third_party/http_parser, is this going to be the final directory for third party libraries for http? I see another 3rd party directory here https://github.com/aws/aws-iot-device-sdk-embedded-C/tree/development/libraries/3rdparty.

Reply:
This is not shared so we put it here.